### DISCUSSION TITLE
Should <PackageFile> accept modifiers such as BuildAction or Flatten ?

### DISCUSSION THREAD

**Discussion by verquepasa** [devlooped/nugetizer#116] [Q&A] [answered]
not much to add, I thought those options belong there

**Comment by kzu** [comment, marked as answer]
For package files with PackFolder=contentFiles, this is already supported: https://github.com/devlooped/nugetizer/blob/main/src/NuGetizer.Tasks/CreatePackage.cs#L344. This also applies to Content items.

Documentation doesn't mention this, however, but it should just work.

**Reply by kzu** [reply]
The mapping of PackFolder to PackageFolder is done at https://github.com/devlooped/nugetizer/blob/main/src/NuGetizer.Tasks/AssignPackagePath.cs, with the default values for the folder kind being specified in https://github.com/devlooped/nugetizer/blob/main/src/NuGetizer.Tasks/NuGetizer.props

**Reply by kzu** [reply]
This is now properly documented too https://www.clarius.org/nugetizer/#contentfiles
